andela/ah-frontend-thanos
# #162693578 Make a better first impression on the homepage

#### What does this PR do?
Adds a carousel and CSS styling to the homepage
#### Description of Task to be completed?
- Add homepage carousel below the navigation bar
- Show actual article images in the footer (remove placeholder images)
- Centre the "Read More" button for each article
- Add the theme colour (turquoise) where appropriate
#### How should this be manually tested?
- Go to the homepage
- There should be a carousel that scrolls automatically
- Click the article title to go to that article's page
- The footer should have article images from actual articles
- Click an image in the footer to go to article
- The "Read More" button for each article should be centred.
- The "Top" button in the footer should scroll to the top
#### What are the relevant pivotal tracker stories?
[#162693578](https://www.pivotaltracker.com/story/show/162693578)
#### Screenshots
*Homepage carousel*
![image](https://user-images.githubusercontent.com/36793346/50093155-fd191800-0220-11e9-9bbd-32731c95e0b4.png)

*Articles on the homepage*
![image](https://user-images.githubusercontent.com/36793346/50093177-0d30f780-0221-11e9-844f-9340bd7c768d.png)

*The footer*
![image](https://user-images.githubusercontent.com/36793346/50093204-19b55000-0221-11e9-9271-caf5558aec2e.png)
